totalRP3/Modules/Register/Main/RegisterList.lua

```diff
@@ -1045,7 +1053,11 @@ TRP3_API.RegisterCallback(TRP3_Addon, TRP3_Addon.Events.WORKFLOW_ON_LOADED, func
 			end,
 			onClick = function(unitID)
 				openMainFrame();
-				openPageByUnitID(unitID);
+				if unitID == Globals.player_id then
+					TRP3_API.navigation.menu.selectMenu("main_10_player");
```

Comment:
I'm not quite sure what that was supposed to do, I imagine it was meant to be modified so it wouldn't reselect the player menu if that was already the active menu, but right now it just does the exact same thing openPageByUnitID was doing when unitID was the player.

Reply:
This wasn't meant to go in, as it does not do anything indeed. Sorry for that, will remove in 'future' push.